lib/config.ts

```diff
@@ -47,6 +49,7 @@ type ResolvedConfig = {
   experiments: Experiment[];
   mockedIP?: string;
   sessionID: string;
+  skipEnrichment: boolean;
```

Comment:
This doesn't need to be a forced type. `undefined` is falsy, which is the default behavior here. 
```suggestion
  skipEnrichment?: boolean;
```